kythe/go/platform/tools/kzip/mergecmd/mergecmd.go

```diff
@@ -63,9 +63,7 @@ func (c *mergeCommand) Execute(ctx context.Context, fs *flag.FlagSet, _ ...inter
 	if c.output == "" {
 		return c.Fail("required --output path missing")
 	}
-	opts := &kzip.WriterOptions{
-		Encoding: c.encoding,
-	}
+	opts := kzip.WithEncoding(c.encoding.Encoding)
```

Comment:
Name is plural, but is singular.  Maybe:

```
opts := []kzip.WriterOption{kzip.WithEncoding(c.encoding.Encoding)}
...
if err := mergeArchives(ctx, tmpOut, archives, opts...); err != nil {
...
```

Reply:
Just made the name singular.